content/community.html

```diff
@@ -52,6 +52,11 @@ <h1>Contributing</h1>
       <a href="https://groups.google.com/forum/#!forum/prometheus-developers">prometheus-developers</a> – for discussions around Prometheus development
     </p>
 
+    <h1>Releases</h1>
+    <p>
+        Prometheus has a documented release process, to make releases consistent. If you intend to cut a release or want to have a look at the planned release schedule, see the <a href=/releases/>releases</a> page.
```

Comment:
This feels like something that should be linked from the server docs, rather than community

Reply:
What do you mean by the server docs?

Reply:
Everything under https://prometheus.io/docs/prometheus/

Reply:
Seems reasonable. I'll move it.

Reply:
I tried this, without any success. If you have any suggestions on how to do it I'm all open, but I'm out of ideas.

Reply:
It'd have to go in the Prometheus repo.

Reply:
Yes I've thought about that, but that would mean, that we have different/duplicate release schedules and instructions based on the release, or have to duplicate across release branches, none of those options seem like a good idea.

Reply:
We could put it in the documentation directory, it's not as if these docs are relevant to users and need to appear on prometheus.io

Reply:
I think the release schedule would be nice if it was on prometheus.io but I think I prefer the schedule as a nice to have on the website over the current situation. Will do :+1: .